README.md

```diff
@@ -138,9 +138,9 @@ Default:        false
 ```
 Type:           bool
 Valid Values:   true, false
-Default:        false
+Default:        true
 ```
-`allowNativePasswords=true` allows the usage of the mysql native password method.
+`allowNativePasswords=false` disallows the usage of mysql native password method during auth switch process.
```

Comment:
Might be nit-picking, but "during auth switch process" is an internal detail. The average driver user has no idea what is meant by this. Just remove it.
And if you push a change, please also change "mysql" to "MySQL" to keep it consistent within the README